.github/CODEOWNERS

```diff
@@ -0,0 +1,15 @@
+#####################################################
+#
+# List of approvers for OpenTelemetry Node SDK
+#
+#####################################################
+#
+# Learn about membership in OpenTelemetry community:
+#  https://github.com/open-telemetry/community/blob/master/community-membership.md
+#
+#
+# Learn about CODEOWNERS file format:
+#  https://help.github.com/en/articles/about-code-owners
+#
+
+* @justindsmith @draffensperger @iredelmeier @yurishkuro @danielkhan @mayurkale22
```

Comment:
draffensperger is not in the list: https://github.com/open-telemetry/community/blob/master/community-members.md#nodejs

I don't mind as long as you and bogdan are ok. Just make sure to nominate him in community repo

Reply:
I will chat with @mayurkale22 more about my involvement in this tomorrow and should be able to give more clarity soon.

Reply:
I chatted with Mayur today and I'd rather not be added as an approver at this point, because I won't be an active contributor of code to the project and because Mayur already is an involved maintainer from Google for this project.

My interest is primarily around making sure that the library will be compatible with the browser platform (and adding browser instrumentation over time as a side project based on the [opencensus-web](https://github.com/census-instrumentation/opencensus-web) code). I think I can have that influence through code and design reviews without needing to be an official approver.

Reply:
Updated the PR accordingly. PTAL